docs/rfc/rfc-1.adoc

```diff
@@ -184,22 +184,33 @@ reason), all they need to do is not submit their reveal and wait 30 days.
 
 [[sweeping]]
 ==== Sweeping
+
+Governable Parameters
+
+- `sweeping_refund_safety_time`: The amount of time prior to when a UTXO
+  becomes eligible where we will not include it in a sweeping transaction.
+- `sweep_period`: The amount of time we wait between sweeps.
+- `sweep_max_btc`: The amount of total bitcoin that will trigger an early sweep.
+- `dust_threshold`: The minimum bitcoin deposit amount for the transaction to
+  be included in a sweep.
```

Comment:
Should we also include as a governable param the amount of time after which we start bumping up the fee for the Bitcoin sweeping transaction?

Reply:
Yeah - I'll include these params, but in a different PR since the current sweeping section doesn't reference fee increases and I want to scope this PR to just the formatting refactor

Reply:
alrighty - included in https://github.com/keep-network/tbtc-v2/pull/65